### ISSUE TITLE
Move dataset status badges, and add information about them.

### ISSUE THREAD

**Issue by glatard** [CONP-PCNO/conp-portal#319]
## Purpose

The dataset status badges, located in the main dataset page, are a bit prominent and might not be well understood by users not familiar with continuous testing of dataset availability. After discussing with @cmadjar, @emmetaobrien and @samirdas, we agreed on the following things to address this issue:

1. Remove status badges from the main dataset page, leave them only in the dataset details page. In the dataset details page, move them next to the "Dataset download instructions" title. 
2. Next to the status badge, add a "question mark icon" with the following hover text:

> The CONP integrates datasets stored in multiple platforms, to accommodate the various governance models adopted by diverse research groups. While this decentralized architecture nicely fits with open-science goals, it also has challenges, as the availability of dataset storage locations may fluctuate. For this reason, CONP datasets are periodically tested (every few hours), to ensure a satisfying user experience. This status badge shows the status of the dataset as returned by the latest iteration of the tests. In general, it gives a good indication whether data download can expected to succeed. In some cases, however, status may flicker due to temporary network glitches, transient misconfiguration or other issues. In any case, the CONP team is working hard to ensure the high-availability of the datasets present in the platform!

@samirdas, @emmetaobrien, @cmadjar, feel free to reword this as you see fit.

## Context

Status badges are both important and potentially misleading. The proposed modification would make them less prominent to people who don't intend to download any data, while providing more explanation to those who will.

## Possible Implementation (optional)

See description above.

**Comment by jbpoline** [comment]
:+1:

**Comment by cmadjar** [comment]
@liamocn could you look at Tristan's description above and implement it? This is very well detailed. Thanks @glatard! Sorry I did not follow up on this, it fell off of my radar somehow... 

Let me know when you have a PR ready and I will gladly test it :)

**Comment by liamocn** [comment]
Will do, cheers!
